.github/workflows/checkAddonUrlAndHash.js

```diff
@@ -0,0 +1,96 @@
+const glob = require('glob');
+const crypto = require('crypto');
+const { v4: uuidv4 } = require('uuid');
+const fs = require('fs');
+const { exec } = require('child_process');
+const PROJECT_URL = "https://github.com/nvaccess/addon-datastore/blob/master/";
+
+function removeMetadataFile({core}, metadataFile, reason) {
+  console.log(`Deleting file "${metadataFile}" because ${reason}`);
+  // normalise the Windows-style path to be usable for the URL in the PR body
+  const metadataFileNormalised = metadataFile.replace(/\\/g, '/');
```

Comment:
Slightly more self-documenting
```suggestion
  const metadataFileNormalised = metadataFile.replaceAll('\\', '/');
```